src/index.js

```diff
@@ -192,6 +192,13 @@ class Gatherer {
   UnionType(node, context) {
     return [null, {type: "union", typeParams: this.handleNodes(node.types, context, [])}]
   }
+  TypeOperator(node, context) {
+    let type = this.takeUnnamed(node.type, context)
+    // FIXME handle other operators? Need to find out what other values this can have
```

Comment:
The others are `KeyOfKeyword` and `UniqueKeyword`.

Reply:
Ah, that's a useful resource.

By the way, I later found out that the way I'm adding `readonly` to the object to indicate that the _array_ is readonly is somewhat problematic in that we also use that to indicate that a _field_ is readonly.

It should be possible to keep using a single object for these, but we should probably define which properties apply to the binding, and which properties apply to its content type. We could for example use a `readonlyType` property for `readonly x[]`, and keep plain `readonly` for bindings.

(A similar issue comes up when merging data from a getter's return type with data from the property, which I noticed still isn't really right with my patch—we want to take the binding/property fields from the getter, and the type fields from the return type. And if we properly define which is which this'd be a lot simpler.)